aries-site/src/data/structures/learn.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable max-len */
+
+export const learn = [
+  {
+    name: 'How to add search and filter to DataTable with Data',
+    description:
+      'This guide shows you how to use the Data component to easily add searching and filtering to a DataTable.',
```

Comment:
Wondering if we should really embrace the naming of "how-to guide" for clarity and specificity?

```suggestion
      'This how-to guide shows you how to use the Data component to easily add searching and filtering to a DataTable.',
```